gradle-hsql-plugin/gradle.properties

```diff
@@ -0,0 +1 @@
+serverbase = /home/stalele/gradleTomcat
```

Comment:
this property will go away because in  PR-899 we are using build.properties so we dont need to set it in gradle.properties